<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDemo</class>
 <widget class="QDialog" name="DialogDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>0</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    background-color: #0078d4;
    padding: 5px;
    border-radius:3px;
	min-height:20px;
    color: #ffffff;
    font-size: 20px;
    text-align: center;
}</string>
   </property>
   <property name="text">
    <string>        接收文本</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_openlink_https">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开链接(https)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_openlink_http">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>打开链接(http)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
